Utilities/JIT.cpp

```diff
@@ -316,8 +319,13 @@ jit_runtime_base& asmjit::get_global_runtime()
 	{
 		custom_runtime() noexcept
 		{
+#ifdef _WIN32
+	u64 addr = reinterpret_cast<u64>(GetModuleHandleW(nullptr));
```

Comment:
Either this or `-Wl,--image-base,0x10000` is required for ASMJIT to work. Without it a relocation fails because a number does not fit in a 32-bit integer.

Reply:
It's not a guarantee that the module will always load in low memory though unless I'm mistaken (it probably does, but I haven't found any guarantee by M$). Why not just configure it with the build option?

Reply:
Regardless, the indentation is completely off anyway.

Reply:
reverted

Reply:
Can you fix it in ASMJIT?

Reply:
Sorry I'm not qualified enough. And it works with `-Wl,--image-base,0x10000` anyway. That's good enough for me.